styles/lemmy/catppuccin.user.css

```
@@ -0,0 +1,229 @@
+    @mocha:     { @rosewater: #f5e0dc; @flamingo: #f2cdcd; @pink: #f5c2e7; @mauve: #cba6f7; @red: #f38ba8; @maroon: #eba0ac; @peach: #fab387; @yellow: #f9e2af; @green: #a6e3a1; @teal: #94e2d5; @sky: #89dceb; @sapphire: #74c7ec; @blue: #89b4fa; @lavender: #b4befe; @text: #cdd6f4; @subtext1: #bac2de; @subtext0: #a6adc8; @overlay2: #9399b2; @overlay1: #7f849c; @overlay0: #6c7086; @surface2: #585b70; @surface1: #45475a; @surface0: #313244; @base: #1e1e2e; @mantle: #181825; @crust: #11111b; };
+  };
+   /*  Userstyle as Mixin
+       *  This takes the following Parameters:
+       *  @lookup – which is the desired Flavor
+       *  @accent – which is the desired Accent Color */
+		#catppuccin(@lookup, @accent) {
+			@rosewater: @catppuccin[@@lookup][@rosewater];
+			@flamingo: @catppuccin[@@lookup][@flamingo];
+			@pink: @catppuccin[@@lookup][@pink];
+			@mauve: @catppuccin[@@lookup][@mauve];
+			@red: @catppuccin[@@lookup][@red];
+			@maroon: @catppuccin[@@lookup][@maroon];
+			@peach: @catppuccin[@@lookup][@peach];
+			@yellow: @catppuccin[@@lookup][@yellow];
+			@green: @catppuccin[@@lookup][@green];
+			@teal: @catppuccin[@@lookup][@teal];
+			@sky: @catppuccin[@@lookup][@sky];
+			@sapphire: @catppuccin[@@lookup][@sapphire];
+			@blue: @catppuccin[@@lookup][@blue];
+			@lavender: @catppuccin[@@lookup][@lavender];
+			@text: @catppuccin[@@lookup][@text];
+			@subtext1: @catppuccin[@@lookup][@subtext1];
+			@subtext0: @catppuccin[@@lookup][@subtext0];
+			@overlay2: @catppuccin[@@lookup][@overlay2];
+			@overlay1: @catppuccin[@@lookup][@overlay1];
+			@overlay0: @catppuccin[@@lookup][@overlay0];
+			@surface2: @catppuccin[@@lookup][@surface2];
+			@surface1: @catppuccin[@@lookup][@surface1];
+			@surface0: @catppuccin[@@lookup][@surface0];
+			@base: @catppuccin[@@lookup][@base];
+			@mantle: @catppuccin[@@lookup][@mantle];
+			@crust: @catppuccin[@@lookup][@crust];
+			@accent-color: @catppuccin[@@lookup][@@accent];
+
+			[data-bs-theme="dark"] {
+				--bs-body-color: @text;
+				--bs-body-bg: @base;
+				--bs-emphasis-color: @subtext1;
+				--bs-secondary-color: @subtext0;
+				--bs-secondary-bg: @mantle;
+				--bs-tertiary-color: @crust;
+				--bs-tertiary-bg: @crust;
+				--bs-primary-text-emphasis: @overlay0;
+				--bs-secondary-text-emphasis: @overlay1;
+				--bs-success-text-emphasis: @green;
+				--bs-info-text-emphasis: @sky;
+				--bs-warning-text-emphasis: @yellow;
+				--bs-danger-text-emphasis: @red;
+				--bs-light-text-emphasis: @overlay2;
+				--bs-dark-text-emphasis: @overlay2;
+				--bs-primary-bg-subtle: @base;
+				--bs-secondary-bg-subtle: @mantle;
+				--bs-success-bg-subtle: @green;
+				--bs-info-bg-subtle: @@accent;
```

Just quickly reading over i think the linter is failing due to this and others in the file, this is just the first i found.

```suggestion
				--bs-info-bg-subtle: @accent-color;
```